modyn/supervisor/internal/eval_strategies/offset_eval_strategy.py

```diff
@@ -1,49 +0,0 @@
-from typing import Iterable, Optional
-
-from modyn.supervisor.internal.eval_strategies.abstract_eval_strategy import AbstractEvalStrategy
-from modyn.utils import convert_timestr_to_seconds
-
-
-class OffsetEvalStrategy(AbstractEvalStrategy):
```

Comment:
`IntervalEvalStrategy` enables a superset of features of `OffsetEvalStrategy`, therefore i dropped it. If needed, we can also keep this though.